src/components/hide-this-page/default/index.njk

```diff
@@ -0,0 +1,27 @@
+---
+title: Hide this Page
+layout: layout-example.njk
+---
+
+<!-- Code sits above the 'skip to main content' <div> to warn users that this feature exists on the page -->
+<div class="govuk-quick-exit__container govuk-visually-hidden">
+  <p class="govuk-body">
+    To leave the page quickly, press the escape key.
+  </p>
+</div>
+
+<!-- Code used for the component and sits inside the <main> content wrapper -->
+<div class="govuk-quick-exit__container govuk-quick-exit__enabled">
+  <div class="govuk-quick-exit__position">
+    <a href="https://www.google.com" id="quick-exit" tabindex="-1" class="govuk-button govuk-button--warning govuk-quick-exit">
```

Comment:
This link looks like a button - which means it could fail WCAG 2.1 success criteria 1.3.1 - Information and relationships. 

> Sighted users perceive structure and relationships through various visual cues... Having these structures and these relationships programmatically determined or available in text ensures that information important for comprehension will be perceivable to all.

This is a problem because a keyboard user could expect something that looked like a button to be activated with the space bar.

This can functionality can be added using JavaScript, but still remains a problem for those without JavaScript. 

A dummy form could be used around the button, so it would be usable without JS - but then it's not really a semantic use of a form and that could confuse people.

It might just need a slight design tweak to include an underline to emphasise that this is a link rather than a button - not a designer though!

Reply:
I'd assume this could use `role="button"` just like [start buttons do on the design system](https://design-system.service.gov.uk/components/button/#start-buttons). I assume (but you should check) that adding `data-module="govuk-button"` would then correctly shim the behaviour as needed.

Start buttons (and all button links) _also_ don't work with the space bar without js, but I think that's an accepted limitation.

Reply:
Yeah, adding the `data-module` attribute would correctly shim the button-like behaviour. Whilst the space bar not working without JavaScript was an accepted limitation for start buttons, I don't think it should be an accepted limitation for this component - especially as it's a known problem that can be fixed by changing the visual appearance of the component.